x-pack/plugins/lists/server/services/exception_lists/exception_list_client.ts

```diff
@@ -811,9 +827,19 @@ export class ExceptionListClient {
   }: ImportExceptionListAndItemsAsArrayOptions): Promise<ImportExceptionsResponseSchema> => {
     const { savedObjectsClient, user } = this;
 
-    return importExceptionsAsArray({
-      exceptionsToImport,
-      maxExceptionsImportSize,
+    // validation of import and sorting of lists and items
```

Comment:
Can you update the params line on 818 to say that it's an array instead of a stream? That's my fault from the original PR.

Reply:
done 😄